src/pyird/image/aptrace.py

```diff
@@ -210,11 +229,24 @@ def aptrace(dat,cutrow,nap,plot=True):
 
     Returns:
         parameters of a polynomial to trace apertures
+
+    Note:
+        the apertures begin its search at ``cutrow``, which is the row number of the detector (wavelength direction),
+        and continue in the direction of increasing numbers until it matches the appropriate number of apertures.
+        You may as well change the value of ``cutrow`` if the aperture trace is failed.
+        
     """
-    if nap!=42 and nap!=102 and nap!=104:
-        print('Warning: Please set nap = 42 (for H band) or nap = 102 or 104 (for YJ band).')
+    if nap in [42, 102, 104]:
+        print("default nap value")
+    elif nap in [21, 51, 52]:
```

Comment:
Sometimes, it seems that the H-band in single fiber mode only supports `nap=20`. See #107